### DISCUSSION TITLE
元宇宙硬體成主角 台廠推VR眼鏡替代品

### DISCUSSION THREAD

**Discussion by 110409538** [1101-GS4719-C/Python-Community-GS4719-C#493] [科技新知]
台灣的群創光電（3481）在此次CES展中推出「多人用裸眼N3D顯示器」，也就是不需要頭戴裝置、眼鏡，就可以看到3D畫面。這能解決頭戴式頭盔、VR眼鏡等，在長時間配戴時可能造成頭部不適或者暈眩的情況。此外,「多人用裸眼N3D顯示器」具備VR眼鏡的能力，又不需要近距離盯著螢幕，有望成為連結元宇宙的最佳載體。我覺得這個顯示器對需要戴眼鏡或近視度數較深的人來說非常友善。

![image](https://user-images.githubusercontent.com/90551832/159121505-9af7b30b-78f3-4faf-abb6-3dce17dc8ac3.png)

新聞來源:https://tw.news.yahoo.com/ces%E7%98%8B%E4%BB%80%E9%BA%BC2-%E5%85%83%E5%AE%87%E5%AE%99%E7%A1%AC%E9%AB%94%E6%88%90%E4%B8%BB%E8%A7%92-%E5%8F%B0%E5%BB%A0%E6%8E%A8vr%E7%9C%BC%E9%8F%A1%E6%9B%BF%E4%BB%A3%E5%93%81-220000786.html

**Comment by sean1325418** [comment]
文中的可以在遠距離操作家裡機器人技術感覺是可以實現的，用類似VR技術偵測身體動作，再傳訊號給機器人

**Comment by ghost** [comment]
雖然是個不錯的技術構想但感覺離完整體有很大的技術考量需要面對

**Comment by bloodtrail0511** [comment]
未來的工作、娛樂都將在元宇宙中發展，而這個裝置可以不用配戴頭戴裝置，提升環境沉靜感

**Comment by chupochun** [comment]
將生活融入元宇宙，讓生更加精采

**Comment by hapiwangy** [comment]
若真的能夠廣用的話，真的可以方便我們的生活

**Comment by PoooYen** [comment]
感覺還是在產品初期開發階段，感覺如果要實現成商業規模的產業還要再等個10年

**Comment by swerfg03joe** [comment]
看起來雖然很酷炫，但仔細想想播放的節目本身應該就要支援3D，而目前各種節目的製作方式明顯不適合，所以實際用途有多大是值得商榷的。說實話元宇宙只是線上遊戲換個說法而已，到底為甚麼一堆人炒作
